components/authz-service/server/v2/project.go

```diff
@@ -351,7 +351,12 @@ func storageCondition(ruleType storage.RuleType, apiCondition *api.Condition) (s
 		return storage.Condition{}, err
 	}
 
-	return storage.NewCondition(ruleType, apiCondition.Values, condAttr, storage.MemberOf)
+	condOp, err := fromAPIProjectRuleConditionOperators(apiCondition.Operator)
```

Comment:
shorten to `fromAPIConditionOperators`?

Reply:
I agree it's a bit verbose but I was following the convention set already by `fromAPIProjectRuleConditionTypes`.